src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -173,6 +173,22 @@ scans:
 
 #### Suffix
 
+Based on their function, suffixes are divided into three classes:
+
+1. `Suffixes for conventional MRI contrasts`
+
+2. `Grouping suffixes`
+
+3. `Designation suffixes for qMRI maps`
+
+The following three subsections expand on each of these `_suffix` classes.
+
+* **Suffix Class-1: Sufixes for conventional MRI contrasts**
```

Comment:
Note - need to add FLAIR to this grouping and get standard description to capture the clinical use case.